ChangeLog.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to
 
 ### Added
 * (breaking) Some `freud.diffraction.StaticStructureFactorDebye` property names changed to be more descriptive.
+* `freud.locality.PeriodicBuffer` now has an optional `include_input_points` to include the provided system points in the buffer. This defaults to false as to not break behavior.
```

Comment:
```suggestion
* `include_input_points` argument to `freud.locality.PeriodicBuffer`. 
```
If users want all the details on this they can just look it up in the documentation.